forum/qa-plugin/outdated-question-info/frontend/show-outdated-question-info.js

```diff
@@ -18,16 +18,16 @@ if(placeOfOutdatedQuestionInfo){
     if(questionElemExist){
         const outdatedInfoContainerClassList = questionElemExist.classList;
     
-        const areThereAnswersForQuestion = !CKEDITOR.instances.a_content ? true : false;
+        const areThereAnswersForQuestion = !CKEDITOR.instances.a_content;
```

Comment:
Czy nie brakuje tutaj drugiej negacji - `!!CKEDITOR.instances.a_content`? W obecnej formie, jeśli `a_content` jest falsy (czyli np. puste pole edytora z odpowiedzią), to w `areThereAnswersForQuestion` będzie `true` (bo jest pojedyncza negacja), a chyba chcemy odwrotnie. Pisałem o tym w [komentarzu](https://github.com/CodersCommunity/forum.pasja-informatyki.local/pull/297#discussion_r900226390).

Reply:
> Czy nie brakuje tutaj drugiej negacji 
Nie nie brakuje. jeżeli od razu jest wyświetlana instancja CKEditora to wyrażenie zwraca false, ponieważ nie ma przycisku anuluj, więc po co dodawać event listenera do czegoś co nie istnieje?
 Trochę się tutaj jednak pomyliłem przy nazywaniu. Powinno to być bardziej coś w stylu `cancelButtonExist`.